File: iSOLID.RiskAdvisory.App/wwwroot/js/models/dataException.js
leala.models.dataException = function (d) {
    var self = this;

    self.name = ko.observable().extend({ required: true, maxLength: 100 });
    self.code = ko.observable().extend({ required: true, maxLength: 10 });

    self.displayText = ko.pureComputed(function () {
        if (self.name() && self.code()) {
            return self.code() + ': ' + self.name();
        }
        return null;
    });
};

File: iSOLID.RiskAdvisory.Domain/Model/DataException.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace iSOLID.RiskAdvisory.Domain.Model
{
    public class DataException : Entity
    {
        [Required]
        [MaxLength(100)]
        public string Name { get; set; }

        [Required]
        [MaxLength(10)]
        public string Code { get; set; }

        public string DisplayText
        {
            get
            {
                if(string.IsNullOrEmpty(this.Name) || string.IsNullOrEmpty(this.Code))
                {
                    return null;
                }

                return string.Format("{0}: ", this.Code, this.Name);
            }
        }
    }
}
